tests/test_account_page.py

```diff
@@ -22,4 +22,4 @@ def test_logout_user(web_driver: WebDriver):
 def test_shipping_address_setting(web_driver: WebDriver):
     successful_change = my_account_page.shipping_address_setting(web_driver)
     assert True == successful_change, + \
```

Comment:
```suggestion
    assert successful_change, + \
```
Generalnie jak już chcemy to boole porównujemy z `is` a nie z `==`. aleeeeee ogólnie nie powinno się w ogóle wrtzucać w ify i assercje porównywania booli. Wystarczy, że odpowiedź jest `True` albo `False`.